service/src/main.rs

```diff
@@ -157,7 +153,7 @@ fn main() {
 			config.clone(),
 			node_api.clone(),
 			enclave.clone(),
-			DirectClient::new(config.worker_url()),
+			Vec::new(),
```

Comment:
I didn't see any use case where the worker struct would need its own direct enclave server - that's what ECALLs are for. So I replaced it with a `vec` of urls that represents its peers :)

Reply:
I don't  understand. This was the client in case it needs to query other workers on the untrusted side? Although I agree that this field was before we had the trusted server, and we don't have yet a useful rpc on the untrusted side to query.

Reply:
Maybe that was the intention, but it wasn't implemented this way. `config.worker_url()` is its own URL. So it basically was a client to its  own trusted server. And if we want to query other workers, we'd need more than one client. So a vec is necessary anyway. Or did I misunderstand something?

Reply:
I am ok with the peers. I just didn't quite get what you were saying. :)

Reply:
Ah okay. Sorry for being unclear then!